src/services/pcn-transparent-helloworld/src/Transparenthelloworld.cpp

```diff
@@ -70,16 +70,6 @@ void Transparenthelloworld::packet_in(polycube::service::Sense sense,
   }
 }
 
-void Transparenthelloworld::attach() {
```

Comment:
Francesco, not sure we have to *delete* the attach method. To me, attach() makes sense.
What's wrong here is that we ask for the "peer" parameter, which clearly doesn't exist here.
Instead, we should ask for the interface we're attached to, and maybe log it, just to confirm that everything has gone well.

Reply:
Agree with Fulvio, we should have a different implementation for such transparent cube (it is a service cube rather than transparent cube to be more accurate like pcn-synflood), just log and don't have to get the peer parameter.